lib/iris/tests/unit/experimental/ugrid/mesh/test_Mesh.py

```diff
@@ -238,6 +241,7 @@ def test_coords(self):
             {"long_name": "long_name"},
             {"var_name": "node_lon"},
             {"attributes": {"test": 1}},
+            {"location": "node"},
```

Comment:
Why did this need adding? I has assumed that not passing `location` was equivalent to not passing any of the original 3 arguments?

Reply:
I believe this test is set up to loop over this list of kwarg dictionaries and check that each one returns the expected output. Adding an extra dictionary here improves the coverage of this test to include the new argument.

Reply:
Closing after discussion offline